configuration/scripts/ice_in

```diff
@@ -29,14 +29,16 @@
     diagfreq       = 24
     diag_type      = 'stdout'
     diag_file      = 'ice_diag.d'
+    debug_model    = .false.
+    debug_model_step = 999999999
```

Comment:
Would it be better to set debug_model_step to 0 so that it always starts printing immediately, as the default?  Assuming it's ignored unless debug_model=T.

Reply:
It was 99999999 by default before, so I kept it that way.  I'm happy to make the default 0, let me know if you prefer that.

Reply:
I think 0 would be better, since you have a flag to turn it off.

Reply:
Sounds good, I'll make this change in the next test coverage PR.

Reply:
why not this PR?

Reply:
I can do it in this PR too.  I sort of trying to avoid doing some extra testing.  Given this change is minor, nobody is going to use this feature at the moment, and this will be updated on the trunk in the next week (hopefully), I thought I'd just wait.  I have three options
- make the change on this PR and rerun all or a subset of tests
- make the change on the PR and not test
- make the change in the next PR and carry out a full test suite (which I'd do anyway)

I was going for the 3rd option.  I can do the 1st.  I try to avoid doing the 2nd when changes occur in the source code or scripts as it's bad practice and risky.

Reply:
That's fine, whatever works best for you.